pyEpiabm/pyEpiabm/property/place_foi.py

```diff
@@ -2,15 +2,16 @@
 # Calculate place force of infection based on Covidsim code
 #
 
-import pyEpiabm.core
 
 from .personal_foi import PersonalInfection
+from pyEpiabm.core import Parameters
```

Comment:
```suggestion
from pyEpiabm.core import Parameters

from .personal_foi import PersonalInfection
```
Please maintain the PEP8 import order!